tox.ini

```diff
@@ -13,9 +13,13 @@ python =
 max-line-length = 88
 
 [testenv]
+setenv =
+    dirpycobertura = pycobertura/*
 commands =
     pip install -r test-requirements.txt --force-reinstall
-    pytest {posargs} --cov-config=.coveragerc
+    coverage run -m pytest
+    coverage report -m
+    coverage xml
```

Comment:
Oh so was pytest-cov a wrapper to make this easier? Sounds like it!

Reply:
yes